src/chrome/content/rules/m2ch.xml

```diff
@@ -0,0 +1,19 @@
+<ruleset name="M.Dvach">	
+
+	<target host="m2ch.hk" />
+	<target host="www.m2ch.hk" />
+ 	<target host="m2ch.611.to" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586782338#L333
Are you sure m2ch.611.to exist?

Reply:
Yes, it is. It was a DNS issue.